python/chronos/src/bigdl/chronos/forecaster/pytorch.py

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from bigdl.chronos.forecaster.tcn_forecaster import TCNForecaster
+from bigdl.chronos.forecaster.lstm_forecaster import LSTMForecaster
+from bigdl.chronos.forecaster.nbeats_forecaster import NBeatsForecaster
+from bigdl.chronos.forecaster.seq2seq_forecaster import Seq2SeqForecaster
+from bigdl.chronos.forecaster.autoformer_forecaster import AutoformerForecaster
```

Comment:
add a ut file in test/bigdl/chronos/forecaster/test_pytorch.py to make sure these forecasters can be really imported.